### ISSUE TITLE
Is that possible to import the packet data to request header like getting source/destination ip/port via req.Header.Get("SslProxy") ?

### ISSUE THREAD

**Issue by samueljaydan** [sonertari/SSLproxy#67]

**Comment by sonertari** [comment]
Listening program should parse the SSLproxy line in the first packet of a connection to get the source and destination addresses, as explained in README, if I understand your question right.

**Comment by samueljaydan** [comment]
Thanks for the answer. 
Yes you've explained, but I need all packet data, so I can decode other layers also not just the source/destination ip/port. Is that possible to redirect full packet data from your SSLproxy to me. If it is, how to do that?

**Comment by sonertari** [comment]
Yes, that's the purpose of SSLproxy with a listening program. See [the sample lp program](https://github.com/sonertari/SSLproxy/tree/master/tests/testproxy/lp), which is used with e2e tests too.

**Comment by samueljaydan** [comment]
I see, As I understand it, this is redirected by default without adding any config options. just to be able to access this part from the user space.
